Cargo.toml

```diff
@@ -81,6 +82,10 @@ near-token = { version = "0.2.0", features = [
     "interactive-clap",
 ] }
 
+near-abi = "0.4.2"
+zstd = "0.11"
+near-contract-standards = "5.0.0-alpha.2"
```

Comment:
This is a fat dependency, let's avoid it. Just implement or copy ContractSourceMetadata here.

Reply:
Done